src/Sarif.Multitool.Library/Rules/SarifValidationSkimmerBase.cs

```diff
@@ -56,7 +56,7 @@ protected void LogResult(string jPointer, string formatId, params string[] args)
 
             Context.Logger.Log(this,
                 RuleUtilities.BuildResult(
-                    DefaultConfiguration.Level,
+                    null,
```

Comment:
>null [](http://example.com/codeflow?start=20&length=4)

add the name of the variable first:
```variable: null```

with that, we will know what is the variable name that we are passing as `null`

Reply:
added.